tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/CI/GitParserTests.cs

```diff
@@ -6,7 +6,7 @@
 using System;
 using System.Collections.Generic;
 using System.IO;
-using Datadog.Trace.Ci;
+using Datadog.Trace.Ci.CiEnvironment;
```

Comment:
You'll see why shortly... 😛 
```suggestion
using Datadog.Trace.Ci.CiEnvironment;
using FluentAssertions;
```

Reply:
done in [d7bebae](https://github.com/DataDog/dd-trace-dotnet/pull/6259/commits/d7bebae2cda5caa60fd13518156b61df01d04162)